tests/cli/cli-circuit-json-to-tsx.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { expect, test } from "bun:test"
+import { readFile } from "node:fs/promises"
+import { join } from "node:path"
+import { getCliTestFixture } from "../fixtures/get-cli-test-fixture"
+
+test("should convert sample circuit.json to TSX", async () => {
+  const { tmpDir, runCommand } = await getCliTestFixture()
+
+  const inputPath = join(__dirname, "../../assets/input-circuit-json.json")
+  const outputPath = join(tmpDir, "output.tsx")
+
+  // Run the conversion command
+  const { stdout, stderr } = await runCommand(
+    `circuit-json-to-tscircuit ${inputPath} -o ${outputPath}`,
+  )
+  expect(stderr).toBe("")
+
+  // Verify output file content
+  const output = await readFile(outputPath, "utf-8")
+  expect(output).toMatchInlineSnapshot(`
+"import { createUseComponent } from "@tscircuit/core"
+import type { CommonLayoutProps } from "@tscircuit/props"
+const pinLabels = undefined as const
+interface Props extends CommonLayoutProps {
+  name: string
+}
+export const Circuit = (props: Props) => {
+  return (
+    <chip
```

Comment:
@seveibar Is this package `circuit-json-to-tscircuit` only for `Chip` components. As I can see if you pass a simple resistor as the circuit-json in this case, it still outputs a `<Chip />`

Reply:
It should work for different types of components, not always choips

Reply:
The current `convertCircuitJsonToTscircuit` function is just having the `<Chip />` template

https://github.com/tscircuit/circuit-json-to-tscircuit/blob/main/lib/get-component-using-template.ts